homeassistant/components/jvc_projector/icons.json

```diff
@@ -7,6 +7,24 @@
           "on": "mdi:projector"
         }
       }
+    },
+    "sensor": {
+      "jvc_power_status": {
+        "default": "mdi:power-plug-off",
+        "state": {
+          "on": "mdi:power-plug",
+          "warming": "mdi:power-plug-outline",
+          "cooling": "mdi:power-plug-off-outline",
```

Comment:
Can we have better icons for it?
use the outline variant for cooling/warming is wrong in my opinion

Reply:
`heat-wave` and `snowflake` look like good choices to me.